src/components/v5/frame/ColonyHome/partials/FundsCards/FundsCards.tsx

```diff
@@ -24,39 +22,21 @@ export const FundsCards = () => {
     });
   };
 
-  const { colony } = useColonyContext();
+  const subTeams = useSubDomains();
 
-  const teams = colony.domains?.items || [];
-
-  // Teams always will have at least 1 team by default - it is general
-  const isAddNewTeamVisible = teams.length < 2;
+  const isAddNewTeamVisible = !subTeams || subTeams.length < 1;
   return (
     <div className="flex flex-col gap-4 sm:flex-row sm:gap-2">
       <WidgetCards.List className="w-full">
         <FundsCardsTotalItem />
-        {teams.map((item) => {
-          const currency = 'USD'; // @TODO: use item currency instead
-          return (
-            <WidgetCards.Item
-              key={item?.id}
-              // @TODO: update onClick when it will be ready
-              onClick={() => {}}
-              title={item?.metadata?.name}
-              // @TODO: update with a real teams data
-              subTitle={
-                <FundsCardsSubTitle
-                  currency={currency}
-                  value={
-                    <Numeral
-                      value="15,779.00"
-                      prefix={currencySymbolMap[currency]}
-                    />
-                  }
-                />
-              }
-            />
-          );
-        })}
+        {subTeams?.map((item) => (
```

Comment:
maybe we could first filter out the null items, amplify is pretty annoying regarding that, so maybe a `.filter(notMaybe)` before the map